teams-notifier-console-plugin/src/main/resources/teams/teamsNotifierConfiguration.jsf

```diff
@@ -118,7 +119,7 @@ holder.
                           label="$resource{i18nemn.notifier.teams.configuration.webhookUrlLabel}"  
                           helpText="$resource{i18nemn.notifier.teams.configuration.webhookUrlLabelHelpText}">
                 <sun:textField id="webhookUrlField" columns="$int{75}" maxLength="512" 
-                               text="#{pageSession.valueMap['webhookUrl']}" styleClass="required"  
+                               text="#{pageSession.webhookUrl}" styleClass="required"
```

Comment:
This breaks the admin console in a different way.

Go to the admin console and set the webhook URL to something and save it - all seems gravy.
Click away and come back to it, webhook URL field is now empty - hmm....
Execute `get-teams-notifier-configuration` asadmin command - webhook URL is set to what you set it to.

So while you've fixed the setting of the property from the admin console, you've now broken the retrieval of said property.